### ISSUE TITLE
Setting a Random seed to ensure Linnea generates the same set of variants for repeated generations 

### ISSUE THREAD

**Issue by as641651** [HPAC/linnea#32]
While using the script 'examples/run_linnea.py' for a random equation over and over again, is it possible to set a seed so that Linnea generates the **same set of variants in the same order** every time?

**Comment by hbarthels** [comment]
Yeah, I think so. Have you tried setting `random.seed(0)` (of course the value doesn't need to be `0`) before generating the random equation?
